app/src/main/kotlin/co/netguru/baby/monitor/client/common/base/BaseFragment.kt

```diff
@@ -1,11 +1,11 @@
 package co.netguru.baby.monitor.client.common.base
 
 import android.os.Bundle
-import android.support.v4.app.Fragment
+import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.ViewGroup
 
-abstract class BaseFragment: Fragment() {
+abstract class BaseFragment: androidx.fragment.app.Fragment() {
     abstract val layoutResource: Int
 
     override fun onCreateView(
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/3c5401c2c1e764397645482fac20357348df064f/app/src/main/kotlin/co/netguru/baby/monitor/client/common/base/BaseFragment.kt#L11"></span>Parameter should be on a separate line (unless all parameters can fit a single line)</td>
    </tr>
  </tbody>
</table>